src/test/java/com/github/javafaker/PokemonTest.java

```diff
@@ -16,4 +16,9 @@ public void name() {
     public void location() {
         assertThat(faker.pokemon().location(), matchesRegularExpression("\\w+( \\w+)?( \\w+)?"));
     }
+
+    @Test
+    public void type() {
+        assertThat(faker.pokemon().location(), matchesRegularExpression("\\w+( \\w+)?( \\w+)?"));
```

Comment:
this test is wrong, it should be faker.pokemon().type() and not location()

Reply:
you're right, fixed into https://github.com/DiUS/java-faker/pull/775/commits/f319dd1f9e4cbe209d8aa260018d11af4a34766d